docs/project/2024/raul_anton.jac/index.md

```diff
@@ -0,0 +1,94 @@
+# MeloLED
+MP3 Player with LED strip using Pico RP2040
+
+:::info 
+
+**Author**: Raul-Anton Jac \
+**GitHub Project Link**: [https://github.com/jacraul/meloled](https://github.com/jacraul/meloled)
```

Comment:
```suggestion
**GitHub Project Link**: https://github.com/jacraul/meloled
```

I don't think this is the project's repository, it has to be the one that was created when you accepted the Project assignment.